File: engine/engine-model/src/main/scala/net/server/channel/packet/cash/operation/IncreaseCharacterSlotsPacket.scala
package net.server.channel.packet.cash.operation

class IncreaseCharacterSlotsPacket(private var _action: Int, private var _cash: Int, private var _itemId: Int) extends BaseCashOperationPacket(_action = _action) {
  def cash: Int = _cash

  def itemId: Int = _itemId
}

File: engine/engine-model/src/main/java/net/server/channel/packet/cash/operation/IncreaseCharacterSlotsPacket.java
package net.server.channel.packet.cash.operation;

public class IncreaseCharacterSlotsPacket extends BaseCashOperationPacket {
   private final Integer cash;

   private final Integer itemId;

   public IncreaseCharacterSlotsPacket(Integer action, Integer cash, Integer itemId) {
      super(action);
      this.cash = cash;
      this.itemId = itemId;
   }

   public Integer cash() {
      return cash;
   }

   public Integer itemId() {
      return itemId;
   }
}
